configure.ac

```diff
@@ -117,30 +92,6 @@ if test "$with_selinux" != "no"; then
 fi
 AM_CONDITIONAL([WITH_SELINUX], [test "x$with_selinux" = "xyes"])
 
-if test "$prefix" = "/usr" && test "$sysconfdir" = '${prefix}/etc'; then
-	sysconfdir="/etc"
-fi
-if test "$prefix" = "" && test "$datarootdir" = '${prefix}/share'; then
-	datarootdir="/usr/share"
-fi
-if test "$prefix" = "/usr" && test "$localstatedir" = '${prefix}/var'; then
-	localstatedir="/var"
-fi
-if test "x$prefix" = "xNONE"; then
-	prefix="/usr/local"
-fi
-if test "x$exec_prefix" = "xNONE"; then
-	exec_prefix=$prefix
-fi
-SYSCONFDIR=`eval echo $sysconfdir`
-DATAROOTDIR=`eval echo $datarootdir`
-LOCALSTATEDIR=`eval echo $localstatedir`
-BINDIR=`eval echo $bindir`
-AC_SUBST([SYSCONFDIR])
-AC_SUBST([DATAROOTDIR])
-AC_SUBST([LOCALSTATEDIR])
-AC_SUBST([BINDIR])
```

Comment:
These here are necessary for conversion of src/utils/swtpm_conf.h.in and sample/swtpm-localca.conf.in.
If I remove all the `if test` statements above it I get this here in src/utils/swtpm_conf.h:

```
#define SYSCONFDIR "/etc"
#define BINDIR "NONE/bin"
#define DATAROOTDIR "/usr/share"
```

It seems to be that the NONE also needs to be tested for with these test statements. I don't think we can remove this block here.

Reply:
Your right, its still needed. So I split this patch into 2: One to remove the debug checking CFLAGS, and another to m4shify that section